### DISCUSSION TITLE
init ATLAS from contigs or MAGs as well

### DISCUSSION THREAD

**Discussion by Mechah** [metagenome-atlas/atlas#522] [Ideas]
Hi,
as it is possible to skip the qc step in ATLAS if you already have quality filtered reads, I think it would be great to allow ATLAS to initiate a run also from contigs or MAGs and just execute downstream steps of the entire ATLAS pipeline.
What do you think? Or is this already possible?

**Comment by SilasK** [comment]
You can add an argument `genome_dir` with the path to a folder with fast a files (genomes).
Take care to name them with  simple names.

**Reply by Mechah** [reply]
I do not really understand which command could accept the genome_dir as an additional argument. I tried with atlas run genomes and atlas run all but it does not accept this argument. Do I need to specify the genome_dir in the config.yaml file or where can I start the analysis from my genome bins?

**Reply by SilasK** [reply]
Yes you need to add it in the config file.

**Reply by Mechah** [reply]
Hm, but then I get the following error:
AttributeError in line 188 of /home/apps/metagenome-atlas/2.9.1/lib/python3.8/site-packages/atlas/workflow/rules/genomes.smk:
'Logger' object has no attribute 'critical'
  File "/home/apps/metagenome-atlas/2.9.1/lib/python3.8/site-packages/atlas/workflow/Snakefile", line 222, in <module>
  File "/home/apps/metagenome-atlas/2.9.1/lib/python3.8/site-packages/atlas/workflow/rules/genomes.smk", line 197, in <module>
  File "/home/apps/metagenome-atlas/2.9.1/lib/python3.8/site-packages/atlas/workflow/rules/genomes.smk", line 188, in get_genome_dir

Is there something else I need to consider e.g. already with atlas init or the place to the genome_dir in my config file?

**Reply by SilasK** [reply]
For now the limitation is that the genomes should have a .fasta extension.

**Comment by Mechah** [comment]
Cool. ATLAS is getting more and more fun! :-)

**Comment by SilasK** [comment]
# How to use atlas with external genomes

Put all your genomes in a folder. The genomes should be unziped and contain a .fasta extension. 
specify the path to this folder in the config file. 

```
genome_dir: path/to/genome_dir
```

run atlas qc if you don't have quality controlled reads

run 

`atlas run None quant --dryrun` 

And check that no assembly would be performed. 
Activate or deactivate annotations according to your needs. 
(genes are predicted anyway)

**Reply by trickovicmatija** [reply]
The command
```bash
atlas run None quant --dryrun
```
should be changed to :
```bash
atlas run None quantify_genomes --dryrun
```
